ui/StatusQ/src/StatusQ/Components/StatusChatListAndCategories.qml

```diff
@@ -78,194 +75,27 @@ Item {
-            }
-
-            DelegateModel {
-                id: delegateModel
-
-                property int destinationPosition: -1
-
                 model: SortFilterProxyModel {
                     sourceModel: root.model
-
-                    filters: ValueFilter { roleName: "isCategory"; value: true }
-                    sorters: RoleSorter { roleName: "position" }
-                }
-
-                items.includeByDefault: false
-
-                groups: DelegateModelGroup {
-                    id: unsortedItems
-
-                    name: "unsorted"
-                    includeByDefault: true
-                    onChanged: Utils.delegateModelSort(unsortedItems, delegateModel.items,
-                                                       (a, b) => a.position < b.position)
-                }
-
-                delegate: Item {
-                    id: draggable
-                    objectName: model.name
-                    width: statusChatListCategory.width
-                    height: statusChatListCategory.height
-                    property alias chatListCategory: statusChatListCategory
-
-                    StatusChatListCategory {
-                        id: statusChatListCategory
-
-                        property bool dragActive: false
-                        property real startY: 0
-                        property real startX: 0
-
-                        opacity: dragActive ? 0.0 : 1.0
-
-                        dragSensor.drag.target: draggedListCategoryLoader.item
-                        dragSensor.drag.threshold: 0.1
-                        dragSensor.drag.filterChildren: true
-                        dragSensor.onPressAndHold: {
-                            if (root.draggableCategories) {
-                                dragActive = true
-                            }
-                        }
-                        dragSensor.onReleased: {
-                            if (dragActive && delegateModel.destinationPosition !== -1 && statusChatListCategory.originalOrder !== delegateModel.destinationPosition) {
-                                root.chatListCategoryReordered(statusChatListCategory.categoryId, statusChatListCategory.originalOrder, delegateModel.destinationPosition)
-                            }
-                            dragActive = false
-                        }
-                        dragSensor.cursorShape: dragActive ? Qt.ClosedHandCursor : Qt.PointingHandCursor
-                        dragSensor.onPressed: {
-                            startY = dragSensor.mouseY
-                            startX = dragSensor.mouseX
+                    sorters: [
+                        RoleSorter {
+                            roleName: "categoryPosition"
+                            priority: 2
+                        },
+                        RoleSorter {
+                            roleName: "position"
+                            priority: 1
```

Comment:
I believe you can invert the order of the RoleSorters here thus avoiding explicitly specifying their `priority`